lively.graphics/geometry-2d.js

```diff
@@ -771,17 +783,24 @@ export class Rectangle {
 
   partNameNearest (partNames, p) {
     let dist = 1.0e99;
-    var partName = partNames[0];
+    let partName = partNames[0];
 
+    let nearest;
     for (let i = 0; i < partNames.length; i++) {
-      var partName = partNames[i];
+      partName = partNames[i];
       const pDist = p.dist(this.partNamed(partName));
-      if (pDist < dist) { var nearest = partName; dist = pDist; }
+      if (pDist < dist) { nearest = partName; dist = pDist; }
     }
 
     return nearest;
   }
 
+  withWholeNumbers () {
+    const { x, y } = new Point(this.x, this.y).roundTo(1);
+    const { x: width, y: height } = new Point(this.width, this.height).roundTo(1);
+    return new Rectangle(x, y, width, height);
+  }
```

Comment:
May I suggest replacing this new method with a *higher level* `roundTo(n)` for rectangles?
That way we could cover more than just rounding to whole number via this method.

```suggestion
  roundTo (n) {
    const { x, y } = new Point(this.x, this.y).roundTo(n);
    const { x: width, y: height } = new Point(this.width, this.height).roundTo(n);
    return new Rectangle(x, y, width, height);
  }
```

Reply:
Done :heavy_check_mark: